### DISCUSSION TITLE
Bulk adding of records for a given zone

### DISCUSSION THREAD

**Discussion by AlexsLaboratory** [poweradmin/poweradmin#488] [Ideas]
This tool overall has been great, but I was just wondering if there was any thought in the past about adding the feature to add multiple records for a given zone? Very similar to the current ability of adding multiple zones in one go with the "Bulk registration" page.

**Comment by AlexsLaboratory** [comment]
Doing some more digging it looks like this feature was request back in 2017.

**Comment by edmondas** [comment]
This feature will be prioritized, but first I need to examine how the feature could be implemented in our code base with the least amount of hassle.

**Comment by zermus** [comment]
I voted for this, I'd love this feature.

**Comment by edmondas** [comment]
For more details and to track the implementation progress, please refer to issue #306

**Comment by edmondas** [comment]
Feature implemented - please check the referenced implementation task above to see how it will look.
